src/daemon.rs

```diff
@@ -69,10 +69,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
         let mut sys = System::new_all();
         sys.refresh_all();
-        for (pid, process) in sys.processes() {
-            if pid.to_string() == swhkd_pid && process.exe() == env::current_exe().unwrap() {
+        for (pid, _) in sys.processes() {
+            if pid.to_string() == swhkd_pid {
                 log::error!("Swhkd is already running!");
-                log::error!("pid of existing swhkd process: {}", pid.to_string());
+                log::error!("Pid of existing swhkd process: {}", pid.to_string());
                 log::error!("To close the existing swhkd process, run `sudo killall swhkd`");
```

Comment:
Some distributions do not ship `psmisc` in their systems, iirc Void Linux is one example. Maybe just tell the user to "Close the existing swhkd process and try again".

Reply:
Well...if you're on void you probably already know that XD. Just keeping that in there for the new to linux users.